src/components/AppHeaderMenu.vue

```diff
@@ -27,6 +27,13 @@ export default {
 
   methods: {
     clickHandler(idx) {
+      //collapse navbar for mobile view before changing active item.
+      // Click on the button to do so
+      if(window.matchMedia("(max-width: 992px)").matches){
```

Comment:
Check if this isn't for less than or equal medium. 
```suggestion
      if(window.matchMedia("(max-width: 768px)").matches){
```

Reply:
> Check if this isn't for less than or equal medium.
medium is for 992px  and greater ryt? So max-width: 992px ought to do the trick ryt?

Reply:
> > Check if this isn't for less than or equal medium.
> 
> medium is for 992px and greater ryt? So max-width: 992px ought to do the trick ryt?

My bad, that last comment was wrong. Check if it isn't for less than or equal small. I think the toggle comes up when we pass 768px.

Reply:
ah yes, that's correct